ironfish/src/defaultNetworkDefinitions.ts

```diff
@@ -54,120 +54,4 @@ export const TESTING = `{
-        },
-        {
-          "type": "Buffer",
-          "data": "base64:AQAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIDxSx8U8f8AAAAAsA7AHpnPNyNTQRGWgWB9FMZVHtp/TpL9owd22N2JNhut6SRSwXNFHLOYOAZXBCiYpdfkZZmGyz9m7q0AfOKwGpntVIdWrau7YO+oSbMiJwFuBgZKwBeqFjuUb5C+DqL7BGMZyC25Am48+jnM69XqpTVYAJVwBiQZfmF9UGrWZUJzRZQWweZ9gsh6+0WDNTlHj80hNLP6Hd52jvae1a5mYV8/y6iltH93g+oj8LhEM0HShPeUx04adOT0jZuqD9879zDdoFIh3xODu/sLMTCERrjaPyi273mR7TPjbmh6A2vXB/wjt+fQzm9YtVhwgiU2Z9rlLOvQ+e2zfYemJb7WJc4wNs9DrQh9YKPzfzh7h5TSfL/s+ib4quj1Eqacqm/btgHWxSfhKivXPL+bRyo+cAKsK6+3/EHdvYtZLRbi1XM3X6glJqV/Rnkh7U8UGtwm/7JPR0GZqlpnh2rc+umAv0UBN94IBlFi820tTVYRS4DH0IRno0ChJUi9uRD0EZtvqg4SF5FrZfz6ft/sw9eXn5DzuTIDYIkNQmVhbnN0YWxrIG5vdGUgZW5jcnlwdGlvbiBtaW5lciBrZXkwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAJOCWXNwJQ8XDlveCTJaIANbLs+x5aq8Kz9QScq4bvO4RYiDoH2WuV0/siZ4iQzZCin64Wemlbg8jBWU/obRcE"
-        },
-        {
-          "type": "Buffer",
-          "data": "base64:AQEAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAuO2w8b5hxh1lqYwT1PNHS3QDYkLIq97jh8B0n0adIvKfwwYJwa7o8tdbvIxtYlvep/0YghAjTRhjxdQBP1P8hGcKcCqm3x3L4zLjGVQRU6RiImMGVCZgP1Lh2yLUJCCsDpTT7YWcdipmkwGr0mzgPx72ST8BxtBiKItP+S4UHsghV/yPhns/ol7IWTHIMJUmh3Ms3utcOQDf1G83spzLY5he8MZ+ZzoLj2GrWRe5UjUm0UdUy+7gQ+rnoNvEooA0bIjhPELqIz+UCtMI8ibWp3oxlkO8VVfKH4kuDl4bHiO8POZ5NglD2d3kZRE7X0rRLknPjInWrZwXixcyaxBFZ8r66sP+7yOMa/ow5VcxJl7rqExprGZdrb0ZMlAAm89iAgAAAPWzEPHGPIdH8vAMW7IDfNCiW4QjG9FIgPSv4Vl9VcMEl/h9waGqQ0+69d00YkvX5B+d9Qeep2+Zh6uNcxoEWMq9BWI9F3X39nDSdrm+3L+BKFr6HYE3Kf6WFiJvj820BbKPGmV5AGX4rACeX66QdVwAOjFjRpFCpCltxyJeMSC/d2Fn7rGHHpisF8x2FhI5y6gr0SJW9J0AoN88sgm+ycMGiDYzdNrLvL6cPoo46AO89GcwANz1Oqo7wEU4s5M4uATV7K07PvU+W4aPXr85wdUI0kmcTPyKsR9HKjJbhp6R9k1+nUGTpjfRHFIyZ44wwY2i+iXv0uZHY/nfC4NXOABGn7K5xjJvFtlccOYOq4O2Ixo13MnxZ1JTvT42N9+fAunclLXYyr9RJE+m3L7WWo65GVFXyEo6LHYrDh2c+yWSYbSwd5kOEDYuiF/kat0v91qLANYSgAOBail7df0MoCrKZp7VchsDu+yKLZkMY/J3g8MdgnRs1mvIEbOT9EPA2HpN51/mJPaSwG40PsDrb4iT9AD8uT0FPCjwxWysezk9N6XC5/LEfx9XYyDdZieOFldiJa2M6ptXLZw21isIwm4dBCxO5VrjUfK/uf1WWGEZmTsAWci79LwLlDvzx+myL+Jj78uGzIb7BnNimoZjPWw1tbaW8y+tiBHgSBlMAgb6Q/3bpvA8WyLgUGtC4VtC9LjP7cXRQSwLE7EgLnw+4epKS4QB0Narqad81A8PMPoO2Y2rCS2GIMaDfz2+q+8mpiNa2NL3jtMN63st8jrx9V0QlGK3D3zbl4EmUhCXdd6dWtHajGFKNQydaxF1ua7CSM5iy/tBRSXiC+2AyLY16Rp4MCzzPepLAw=="
-        }
-      ]
-    },
-    "consensus": {
-        "allowedBlockFutureSeconds": 15,
-        "genesisSupplyInIron": 42000000,
-        "targetBlockTimeInSeconds": 60,
-        "maxSyncedAgeBlocks": 60,
-        "targetBucketTimeInSeconds": 10,
-        "maxBlockSizeBytes": 2000000
-    }
-}
- `
-
-export const DEV = `
```

Comment:
Can we bring the dev network back?

Reply:
Yeah I think the only network that should be deleted is the one for testnet phase 2.

Reply:
Yep I can add this back